lib/grp.gd

```diff
@@ -709,8 +710,10 @@ InstallTrueMethod( IsSimpleGroup, IsSporadicSimpleGroup );
 DeclareProperty( "IsAlmostSimpleGroup", IsGroup );
 InstallTrueMethod( IsGroup, IsAlmostSimpleGroup );
 
-# a simple group is almost simple
-InstallTrueMethod( IsAlmostSimpleGroup, IsSimpleGroup );
+# only a nonabelian simple group is almost simple
+#InstallTrueMethod( IsAlmostSimpleGroup, IsSimpleGroup ); # WRONG!
+InstallTrueMethod( IsAlmostSimpleGroup, IsSporadicSimpleGroup );
```

Comment:
Technically this line is redundant, since a known sporadic simple group is simple and perfect via the above `InstallTrueMethod`s, and is therefore almost simple via the below `InstallTrueMethod`. Could be removed, although also no harm in keeping it I suppose.

Reply:
Good point, done.